core/src/components/checkbox/checkbox.ionic.scss

```diff
@@ -1,34 +1,46 @@
-@import "./checkbox";
-@import "./checkbox.ionic.vars";
+@use "../../themes/ionic/ionic.globals.scss" as globals;
 
 // Ionic Checkbox
 // --------------------------------------------------
 
 :host {
   // Border
-  --border-radius: #{$checkbox-ionic-border-radius};
-  --border-width: #{$checkbox-ionic-border-width};
-  --border-style: #{$checkbox-ionic-border-style};
-  --border-color: #{$checkbox-ionic-background-color-off};
+  --border-radius: #{globals.$ionic-border-radius-100};
```

Comment:
Let's remove this since the default soft has been added back.

Reply:
removed!